src/Microsoft.DotNet.Cli.Utils/PathUtility.cs

```diff
@@ -198,5 +199,43 @@ public static string GetPathWithDirectorySeparator(string path)
                 return GetPathWithBackSlashes(path);
             }
         }
+
+        public static string GetProjectRootPath(string path = null)
+        {
+            var directoryPath = Directory.GetCurrentDirectory();
+
+            if (path != null)
+            {
+                try
+                {
+                    directoryPath = Path.GetDirectoryName(directoryPath);
```

Comment:
should this be directoryPath = Path.GetDirectoryName(path); ?

seems like is not being used anywhere